### PR TITLE
Update axi adc to new buffer functions

### PR DESCRIPTION
**Pull request by mchindri**
It was tested on ad9361

### CODE REVIEW

File: projects/ad9081/src/app_iio.c [hunk 1]
```diff
@@ -56,6 +56,11 @@
 /************************** Functions Implementation **************************/
 /******************************************************************************/
 
+struct iio_data_buffer g_read_buff = {
+		.buff = (void *)ADC_DDR_BASEADDR,
+		.size = 0xFFFFFFFF,
```
Comment (amiclaus) [inline projects/ad9081/src/app_iio.c:61]: the `size` value seems to be propagated throughout all projects that support IIO. is it possible to add a macro for this?

**Comment by mchindri** [comment]
v2:
 - Astyle

**Review by cristiantpop — APPROVED** [review]

**Comment by mchindri** [comment]
v3:
 - More astyle

File: projects/ad9081/src/app_iio.c (hunk 1 above)
Comment (mchindri) [reply projects/ad9081/src/app_iio.c:61]: I think I can change this if needed in a other PR for both ADC and DAC. Will make more sense together.

**Review by amiclaus — APPROVED** [review]

**Pull request merged**